docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       - ACCESS_KEY_ID=s3manager
       - SECRET_ACCESS_KEY=s3manager
       - USE_SSL=false
+      - BASE_PATH=myPath
```

Comment:
The Docker compose setup should be as minimalist as possible. Therefore, we shouldn't add customization of the base path here.